catalogue/webapp/components/WorksSearchResults/WorksSearchResults.tsx

```diff
@@ -10,14 +10,27 @@ type Props = {
   apiProps: CatalogueWorksApiProps;
 };
 
+const SearchResultUnorderedList = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-wrap: wrap;
+  padding: 0;
+  margin: 0;
+`;
+
+const SearchResultListItem = styled.li`
+  flex-basis: 100%;
+  max-width: 100%;
+`;
+
 const WorkSearchResults: FunctionComponent<Props> = ({
   works,
   apiProps,
 }: Props) => {
   return (
-    <ul className={'grid plain-list no-padding no-margin'}>
+    <SearchResultUnorderedList>
```

Comment:
I _think_ removing the `no-margin` class might have the same effect  – it was recently made `!important` (because if we want no-margin, we definitely want no-margin – turns out we didn't really want that here).

Reply:
How bizarre, yes it does! I think my preference would still be for styled-components stuff here as we don't really need the full power of the responsive grid - what do you think?

Reply:
The `no-margin` change has highlighted a couple of places where we've not really wanted no-margin but it was previously being overridden anyway. It's worth knowing about in case there are any other things that appear to have gone weirdly out of whack in the last few weeks.